src/components/animate/animate.jsx

```diff
@@ -52,6 +54,7 @@ class Animate extends React.PureComponent {
 Animate.propTypes = {
   children: PropTypes.node,
   className: PropTypes.string,
+  animationName: PropTypes.string.isRequired,
```

Comment:
Can you add a javadoc comment above this line to say that this needs to be set for the component to render anything. (This will be converted to a comment in styleguidist). Like so:

`/** This needs to be set in order for the component to render anything */`